deployability/modules/testing/tests/test_agent/test_registration.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from ..helpers import utils
+from ..helpers.constants import CLIENT_KEYS
+import re
+import json
+
+@pytest.fixture
```

Comment:
```suggestion

@pytest.fixture
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4998/commits/db29c08420c01890d8ca7fedddeb25b94ab07fc3